firmware/shared/controls/tvFactor.h

```diff
@@ -1,35 +1,39 @@
 #pragma once
-#include <vector>
 #include <algorithm>
 #include <tuple>
 #include "../util/mappers/lookup_table.h"
 
 namespace ctrl{
 
-template <typename T>
-T EnableTorqueVectoring(T steering_angle, T torque_vectoring_switch) {
-    if (torque_vectoring_switch > 0)
-        return steering_angle;
-    else return 0;
+bool EnableTorqueVectoring(bool torque_vectoring_switch) {
```

Comment:
why have this function? it just returns its input

Reply:
That's exactly what I thought to be honest, but I wasn't sure if I should omit it since we talked previously about making the function a bool